Sources/HealthCheck/Health.swift

```diff
@@ -0,0 +1,79 @@
+// Health check API
+// see https://inadarei.github.io/rfc-healthcheck/ for more info
+import Vapor
+
+// MARK: Health
+// Structs for generating the Health API response
+public struct Health {
+    // Health status indication options
+    enum Status: String {
```

Comment:
You can make this conform to `Equatable` as well so you don't have to compare on `rawValue`

Reply:
Nice :D I have actually been trying to avoid that.